test/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+[sh_test(
+    name = "{test}".format(test = test),
+    # top showed me 50-400mByte of usage, so "enormous"...
+    #
+    # https://bazel.build/reference/be/common-definitions#test.size
+    size = "enormous",
+    srcs = ["bazel_test.sh"],
+    args = [],
+    data = [
+        "//:openroad",
+    ] + glob(["**/*"]),
```

Comment:
that is a big glob :) I guess that is for testing now, but of course being more specific would be good.

Reply:
I'll do a pass of squash and refine this PR before it is ready for review.

Trying now `bazel test -c opt aes_nangate45`

Reply:
@maliberty I've started at my share of detailed routing logs, I have not seen it stuck where it is stuck below. Is that indicative that I'm not running an optimized openroad build?

```
[INFO DRT-0036] via8 guide region query size = 0.
[INFO DRT-0036] metal9 guide region query size = 12.
[INFO DRT-0036] via9 guide region query size = 0.
[INFO DRT-0036] metal10 guide region query size = 0.
[INFO DRT-0179] Init gr pin query.
[no more output for minutes]
```

Reply:
It completed, timed out in 3600 seconds before. so `bazel test -c opt ...` is at least 6x faster.

```
INFO: Build completed successfully, 4497 total actions
//test:aes_nangate45                                                     PASSED in 561.8s

Executed 1 out of 1 test: 1 test passes.
```